SECURITY.md

```diff
@@ -5,9 +5,9 @@
 
 | Version  | Supported          |
 | -------- | ------------------ |
-| 42.3.x   | security fixes, features, bug fixes |
+| latest 42.x | security fixes, features, bug fixes |
 | 42.2.x   | (the latest branch that supports Java 6, and 7): security fixes, critical bug fixes only. |
-| < 42.2.x | security fixes (upon request) |
+| all the other versions | security fixes (upon request) |
```

Comment:
Sadly this is not clear enough, I interpret this more like "all the older versions", yet this is also confusing since it's unclear how far the "all" can go, if that means that if a user request a security fix for 8.0-325 we should provide it?

I think we should limit the scope to something like branch > 9.2 (which is BTW the first version released to maven) or even > 9.4.

Everything else should not be supported even if contains a critical security issue (and yes, AFAIK I'm not aware that PostgreSQL supports older versions even if they contain security issues).

Reply:
What do you think of `the rest versions` instead of `all the other versions`?

Reply:
I think it would be reasonable to say "some previous versions (within  reason)"

Reply:
I like that wording, it's not open to all versions and it's clear that it's a discretionary policy.

Reply:
I'm inclined to merge this the way it is unless there are objections?

Reply:
I'm inclined we have to support each version for a certain time, otherwise, people have no clue when they need to upgrade.

Reply:
Ya, I can see that logic. so we need to create a security release for 42.3.x then ?

Reply:
I think so 🤷‍♂️ .

Even though the upgrade to 42.4 should not be problematic, I can easily imagine that the ones who upgrade in a hurry would prefer to have "just CVE fix" rather than "bump from 42.2 to 42.8.3"